tfhe-benchmark/benches/integer/vector_find.rs

```diff
@@ -0,0 +1,217 @@
+use benchmark::params_aliases::BENCH_PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128;
+use benchmark::utilities::{write_to_json, OperatorType};
+use criterion::{black_box, criterion_group, Criterion};
+use tfhe::integer::keycache::KEY_CACHE;
+use tfhe::integer::{IntegerKeyKind, RadixClientKey};
+use tfhe::keycache::NamedParam;
+use tfhe::shortint::AtomicPatternParameters;
+use tfhe::MatchValues;
+
+pub fn match_value(c: &mut Criterion) {
+    let bench_name = "integer::match_value";
+
+    let param = BENCH_PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128;
+    let atomic_param: AtomicPatternParameters = param.into();
+    let param_name = param.name();
+
+    let (cpu_cks, cpu_sks) = KEY_CACHE.get_from_params(atomic_param, IntegerKeyKind::Radix);
+
+    //
+    // Latency test: 64 bits, 10 elements
+    //
+    {
+        let num_elements = 10;
+        let bits = 64;
+        let num_blocks = 32;
```

Comment:
By looking at this it seems that the two cases 64 bits + 10 elements // 8 bits + 256 elements could be factorized into one for loop.
Unless I'm missing something here.